platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapView.java

```diff
@@ -99,24 +101,28 @@ public class MapView extends FrameLayout implements NativeMapView.ViewCallback {
   @UiThread
   public MapView(@NonNull Context context) {
     super(context);
+    Log.d("MapLibreLibrary", "MapView constructed with context");
```

Comment:
Isn't this library using Timber? If yes, could you use Timber instead of Log?

Reply:
Good catch. Will do.

Reply:
Just pushed an update that uses Timber. Had to add timber and plant it on SDK as it was used in TestApp only, but I think it was worth it.